openassessment/xblock/staff_area_mixin.py

```diff
@@ -375,22 +375,56 @@ def get_student_info_path_and_context(self, student_username, expanded_view=None
         assessment_steps = self.assessment_steps
 
         example_based_assessment = None
+        example_based_assessment_grade_context = None
+
         self_assessment = None
+        self_assessment_grade_context = None
+
         peer_assessments = None
-        submitted_assessments = None
+        peer_assessments_grade_context = []
+
         staff_assessment = staff_api.get_latest_staff_assessment(submission_uuid)
+        staff_assessment_grade_context = None
+
+        submitted_assessments = None
+
+        grade_details = None
+
+        workflow = self.get_workflow_info(submission_uuid=submission_uuid)
+        grade_exists = workflow.get('status') == "done"
```

Comment:
It doesn't look like it-- I see a lot of places in the code where the workflow statuses are compared/iterated through, and not any constants. I'm not feeling particularly inclined introduce constants at this time.